jetty-ee10/jetty-ee10-tests/jetty-ee10-test-integration/src/test/java/org/eclipse/jetty/ee10/test/AliasCheckerSymlinkTest.java

```diff
@@ -69,69 +73,111 @@ private static void delete(Path path)
         IO.delete(path.toFile());
     }
 
-    private static void setAliasChecker(AliasCheck aliasChecker)
+    private static void setAliasChecker(ContextHandler contextHandler, AliasCheck aliasChecker) throws Exception
     {
-        _context.clearAliasChecks();
+        // TODO: why do I need to manually manage the lifecycle???
```

Comment:
Good question!  Why do you?

Reply:
it was a bug in the `HotSwapHandler`. fixed now